pkg/smokescreen/metrics/metrics.go

```diff
@@ -2,20 +2,16 @@ package metrics
 
 import (
 	"errors"
-	"fmt"
 	"net"
-	"sync/atomic"
 	"syscall"
 	"time"
-
-	"github.com/DataDog/datadog-go/statsd"
 )
 
 // metrics contains all of the metric names contained within the smokescreen package.
 // These are used to determine if a given metric name is valid before associating
 // a persistent tag with the metric. This list must be updated with new metric names
 // if the metric should support persistent tagging.
-var metrics = []string{
+var Metrics = []string{
```

Comment:
Accidental export? All usage seems to be in the same package.

Reply:
Ahhh, good catch, fixing now